es-models/gene_centric/gene_centric.mapping.yaml

```diff
@@ -152,6 +154,8 @@ properties:
             properties:
               created_datetime:
                 type: date
+              days_to_treatment:
```

Comment:
I though days_to_treatment had been removed? I'm not sure if this should still be here - someone else would need to confirm. 
https://jira.opensciencedatacloud.org/browse/TT-417

Reply:
also this commit: https://github.com/NCI-GDC/gdc-models/commit/0b9490bd0ea5d8daaa5689cc22dfd7cfa038c6f7#diff-c2f9bca69067af7947739862c15f36f9

Reply:
Thanks for the confirmation. We've reviewed the indices that were built for Lorax, and days_to_treatment is not set on any of the documents. I believe the option to filter on days_to_treatment was also removed from the portal some time ago. So, I think the presence of this field at least isn't visible to the end user. We should still remove it from the mapping, but I don't think it is available anywhere in the portal right now.